src/components/Article.jsx

```diff
@@ -1 +1,30 @@
-// export const Article = () => ();
+import React from 'react';
+
+export function Article({
+  title,
+  date,
+  text,
+}) {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <div className="box">
+      <h2 className="title" data-cy="title">
+        {title}
+      </h2>
+
+      <p className="subtitle">
+        <time dateTime={dateISO} data-cy="date">
+          {dateLocaleString}
+        </time>
+      </p>
+
+      <p data-cy="text">{text}</p>
```

Comment:
```suggestion
      <p data-cy="text">
        {text}
      </p>
```i think it will be more pretty=)

Reply:
agree